server/common/Constants.java

```diff
@@ -19,9 +19,11 @@ public class Constants {
     public static final String TYPEDB_LOG_FILE_NAME = "typedb";
     public static final String TYPEDB_LOG_FILE_EXT = ".log";
     public static final String TYPEDB_LOG_ARCHIVE_EXT = ".log.gz";
-    public static final String DIAGNOSTICS_REPORTING_URI = "https://3d710295c75c81492e57e1997d9e01e1@o4506315929812992.ingest.sentry.io/4506316048629760";
-    public static final String USAGE_STATISTICS_REPORTING_URI = "https://diagnostics.typedb.com/";
+    public static final String ERROR_REPORTING_URI = "https://3d710295c75c81492e57e1997d9e01e1@o4506315929812992.ingest.sentry.io/4506316048629760";
```

Comment:
Sentry is more about errors rather than diagnostics now + we call it `reporting errors` in the config.